src/script/pages/app-publish.ts

```diff
@@ -0,0 +1,139 @@
+import { css, customElement, html, LitElement } from 'lit-element';
+import '../components/app-header';
+import '../components/app-sidebar';
+import '../components/app-card';
+@customElement('app-publish')
+export class AppPublish extends LitElement {
+  constructor() {
+    super();
+  }
+
+  static get styles() {
+    return css`
+      content-header::part(main-container) {
+        display: flex;
+        justify-content: center;
+        padding-top: 5.2em;
+      }
+
+      .header {
+        padding: 1rem 3rem;
+      }
+
+      .header p {
+        max-width: min(100%, 600px);
+      }
+
+      .main-container {
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        justify-items: center;
+        align-items: center;
+      }
+
+      .container {
+        width: min(100%, 1366px);
+        padding: 0 2rem;
+      }
+
+      .container p {
+        max-width: min(100%, 80%);
```

Comment:
Won't this always be 80%? I'm admittedly newer to CSS functions.

Reply:
you are correct this wasn't supposed to be there removed it